contracts/SignerBondsUniswapV2.sol

```diff
@@ -156,10 +180,17 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
     ///      tokens are sent to the asset pool address set during
     ///      contract construction.
     /// @param amount Amount to swap.
```

Comment:
There should be one more param in the description - `riskManager`.

Reply:
https://github.com/keep-network/coverage-pools/pull/88/commits/c70beccb1628146ed51f8dd2cd4b2f9154ca43fb